compiler/src/iree/compiler/Dialect/LinalgExt/IR/LinalgExtOps.td

```diff
@@ -1578,7 +1593,10 @@ def IREELinalgExt_CustomOp : IREELinalgExt_Op<"custom_op", [
   DeclareOpInterfaceMethods<LinalgFusionInterface>,
   DeclareOpInterfaceMethods<ReifyRankedShapedTypeOpInterface>,
   DeclareOpInterfaceMethods<TilingInterface,
-    ["getLoopIteratorTypes"]>
+    ["getIterationDomain",
+     "getLoopIteratorTypes",
+     "getResultTilePosition",
+     "getTiledImplementation"]>
```

Comment:
We put the implementation to [TilingInterfaceImpl](https://github.com/iree-org/iree/blob/main/compiler/src/iree/compiler/Dialect/LinalgExt/Transforms/TilingInterfaceImpl.cpp). Can you move the implementation from LinalgExtOps.cpp to TilingInterfaceImpl.cpp?

Reply:
Oh thats strange. This file shouldnt be in `Transforms` but it should be in `IR`. If someone used just the library built with the files in `IR` that would lead to undefined symbols. I guess we never do that. I moved that file to `IR` folder and moved the implementation into that file.